Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
```

Comment:
### [mock_redis](https://github.com/sds/mock_redis)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.5.4 → 0.31.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/mock_redis/0.5.4/0.31.0)_]





<details>
<summary>Commits</summary>

A change of **406** commits. See the full changes on [the compare page](https://github.com/sds/mock_redis/compare/v0.5.4...v0.31.0).

These are the first 10 commits:
- (f384aca) [Enforce numeric literals separated with underscores](https://github.com/sds/mock_redis/commit/f384aca3f5dd9b80fa50abd9dc66230108c097d8)
- (411aa7b) [Remove unnecessary blank lines](https://github.com/sds/mock_redis/commit/411aa7b1faea20ba2f53fb0d30dbc223c1da17ef)
- (c910cc0) [Enforce spaces after commas](https://github.com/sds/mock_redis/commit/c910cc041afff5d10de84d206d4d827daa3faffa)
- (0e3f8e1) [Enforce spaces around equals in parameter defaults](https://github.com/sds/mock_redis/commit/0e3f8e18d88ea8d4002b96dde54c8b03125003bf)
- (7fd09d6) [Prefer empty literals](https://github.com/sds/mock_redis/commit/7fd09d68f3e923bdebc3188e26be9f361419258f)
- (ba85b62) [Remove unnecessary begin blocks](https://github.com/sds/mock_redis/commit/ba85b62ce77a74085e0b4f99c80d1a6d33fd5588)
- (024b40c) [Indent when statements at same level as case](https://github.com/sds/mock_redis/commit/024b40ca70b07f55738f79828094bfcac1c71e7c)
- (7656262) [Add specific errors to raise_error expectation](https://github.com/sds/mock_redis/commit/7656262b1331f68dae300007be946549def219dc)
- (a64361a) [Fix literal in interpolation](https://github.com/sds/mock_redis/commit/a64361a32316c09e996abbd8eb962be199da0788)
- (2027e58) [Remove unnecessary method call parentheses](https://github.com/sds/mock_redis/commit/2027e584561aa31221fff6fea8d3ad4ad65af1ba)


</details>